.travis.yml

```diff
@@ -1,11 +1,13 @@
 language: ruby
 bundler_args: --without development
 script: bundle exec rspec spec
-env: TIMEOUT=10
+env:
+  - TIMEOUT=10
+  - PROMISCUOUS_BACKEND=bunny
```

Comment:
this will become poseidon once i can get my own Kafka cluster started and the tests running on it